packages/manager-react-components/src/components/templates/onboarding/onboarding.component.tsx

```diff
@@ -103,21 +99,16 @@ export const OnboardingLayout: React.FC<OnboardingLayoutProps> = ({
   img,
```

Comment:
```suggestion
  img = {},
```

To be able to destructure `img` in all cases (solves the breaking test)